lsp-mode.el

```diff
@@ -7191,13 +7217,21 @@ returns the command to execute."
 INITIALIZATION-OPTIONS are passed to initialize function.
 SESSION is the active session."
   (lsp--spinner-start)
+  (hack-local-variables)
```

Comment:
Why this is needed?

Reply:
I'm not sure why its needed but I'll explain the problem it seems to solve. When I would start up emacs fresh and enter a project, the var `lsp-watch-folders` wasn't picked up. I would then `lsp-restart-workspace` and it was seen. I did that as a kludge to get it to pick them up but not sure why they weren't in the first place. I'll try again though.